frontend/packages/kubevirt-plugin/src/components/vm-disks/disk-summary.tsx

```diff
@@ -6,56 +6,57 @@ import {
   getPVCSourceByDisk,
 } from '../../selectors/vm/selectors';
 import { WINTOOLS_CONTAINER_NAMES } from '../modals/cdrom-vm-modal/constants';
-import { VMKind } from '../../types';
-import { V1Disk } from '../../types/vm/disk/V1Disk';
+import { VMKind, BootableDeviceType } from '../../types';
+import { deviceLabel } from '../boot-order/constants';
 
 const DiskSummaryRow: React.FC<DiskSummaryRowProps> = ({ title, value }) => (
   <div className="kubevirt-disk-summary__disk">
-    <div id={`kubevirt-disk-summary-disk-title-${title}`}>{title}</div>
-    <div id={`kubevirt-disk-summary-disk-value-${value}`}>{value}</div>
+    <span id={`kubevirt-disk-summary-disk-title-${title}`}>{title}</span>
+    <br />
+    <span id={`kubevirt-disk-summary-disk-title-${value}`} className="text-secondary">
+      {value}
+    </span>
   </div>
 );
 
 export const DiskSummary: React.FC<DiskSummaryProps> = ({ disks, vm }) => (
   <div className="kubevirt-disk-summary">
-    {disks.map(({ name }, i) => {
+    {disks.map((disk) => {
+      const { name } = disk.value;
       const container = getContainerImageByDisk(vm, name);
       const pvc = getPVCSourceByDisk(vm, name);
       const url = getURLSourceByDisk(vm, name);
+      const label = deviceLabel(disk);
```

Comment:
please add an argument to this function wether to show the `(CD-ROM)` label or not, and disable it for this particular use case :)

Reply:
done